content/scout/policy/_index.md

```diff
@@ -135,3 +136,23 @@ The list includes the following vulnerabilities:
 - [CVE-2021-44228 (Log4Shell)](https://scout.docker.com/v/CVE-2021-44228)
 - [CVE-2023-38545 (cURL SOCKS5 heap buffer overflow)](https://scout.docker.com/v/CVE-2023-38545)
 - [CVE-2023-44487 (HTTP/2 Rapid Reset)](https://scout.docker.com/v/CVE-2023-44487)
+
+### Supply chain attestations
+
+This policy requires that your artifacts have
+[SBOM](../../build/attestations/sbom.md) and
+[provenance](../../build/attestations/slsa-provenance.md) attestations.
+
+This policy is unfulfilled if an artifact is lacking either an SBOM
+attestation, or a provenance attestation with information about the Git
```

Comment:
condition is backwards here, should be 
> ..., or a provenance attestation _without_ information about the Git repository and base images is being used`

Reply:
It reads to me currently as 
> policy is unfulfilled if ... a provenance attestation with information about the Git repository and base images is being used.

which is the opposite of what we want.

Maybe this?
> This policy is unfulfilled if an artifact is lacking either an SBOM attestation
> or a provenance attestation, or if the provenance is lacking information
> about the Git repository and base images used.

Reply:
your suggestion reads better, let me update it